pkgs/top-level/all-packages.nix

```diff
@@ -17287,6 +17292,7 @@ with pkgs;
   zulu11 = callPackage ../development/compilers/zulu/11.nix { };
   zulu17 = callPackage ../development/compilers/zulu/17.nix { };
   zulu21 = callPackage ../development/compilers/zulu/21.nix { };
+  zulu22 = callPackage ../development/compilers/zulu/22.nix { };
```

Comment:
```suggestion
```

I guess it's safe to drop the line, since only LTS versions of Zulu had Linux support in Nixpkgs and were exposed at the top level. You don't see a `dists.x86_64-linux` in `pkgs/development/compilers/zulu/18.nix`, for example.

If keeping `zulu22` is a necessity, please move these callPackages to `pkgs/top-level/java-packages.nix`.

Reply:
> only LTS versions of Zulu had Linux support in Nixpkgs and were exposed at the top level.

Thanks, I didn't know that, I'm removing `zulu22` from top level